routes/requests.js

```diff
@@ -69,13 +79,28 @@ router.post(path + '/', (req, res) => {
   })
 })
 
+/**
+ * Returns a specific request given a requestId
+ * Response body:
+    * success: Boolean - true if successful, false otherwise
+    * message: String - returns error message if query fails
+    * request: Object - returns a request Object if one is found, null otherwise
+ * Response codes:
+    * 200: Success
+    * 404: Request not found
+    * 500: Something went wrong
+ */
 router.get(path + '/:requestId', (req, res) => {
   const response = {
     success: false,
     message: '',
     request: null
   }
-  getRequest(req.requestId).then((request) => {
+  getRequest(req.params.requestId).then((request) => {
+    if (request === null) {
```

Comment:
Is this enough to verify request is valid? Should we also check if typeof request === 'undefined' ?

Reply:
Nevermind. It checks in the query.